float32 ptx
float32 pty
float32 size
float32 angle
float32 response
int8 octave
int8 class_id
